packages/react/src/runtime/sheet.tsx

```diff
@@ -75,6 +75,8 @@ function lazyAddStyleBucketToHead(bucketName: Bucket, opts: StyleSheetOpts): HTM
     }
 
     const tag = document.createElement('style');
+    // set data-cmpld attribute to "h" for hydrated
+    tag.setAttribute('data-cmpld', 'h');
```

Comment:
The idea here is mainly to tell which style tags were created by the library. Can revert this if needed.